packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -34,6 +35,22 @@ const _showError = remote => alert(_('remoteConnectionFailed'), remote.error)
 const _editRemoteName = (name, { remote }) => editRemote(remote, { name })
 const _editRemoteOptions = (options, { remote }) =>
   editRemote(remote, { options: options !== '' ? options : null })
+const _remoteWithInfo = (remotes, remotesInfo) => {
+  if (!remotesInfo) return remotes
+
+  const remoteWithInfo = {}
```

Comment:
`mapValues` and `map` would probably simplify the implementation, but this one if fine by me.